types/dropin.pp

```diff
@@ -0,0 +1 @@
+type Systemd::Dropin = Pattern['^.+\.conf$']
```

Comment:
I can add a check if there is no slash in there. I tried to use the same kind of pattern definition as for `Systemd::Unit` and there isn't a check for slashes.